assets/src/wizards/subscriptions/index.js

```diff
@@ -40,31 +44,65 @@ class SubscriptionsWizard extends Component {
 	 * Render the example stub.
 	 */
 	render() {
-		const { inputTextValue1, inputTextValue2 } = this.state;
+		const { inputTextValue1, inputTextValue2, currentTask } = this.state;
```

Comment:
- here now also using the `completedTasks`:

```suggestion
		const { inputTextValue1, inputTextValue2, currentTask, completedTasks } = this.state;
```